pom.xml

```diff
@@ -38,17 +38,17 @@
     </licenses>
 
     <properties>
-        <kotlin.version>1.4.32</kotlin.version>
-        <dokka.version>1.4.20</dokka.version>
+        <kotlin.version>1.6.10</kotlin.version>
```

Comment:
Have you managed to check the compatibility between 1.5/1.6?

Reply:
Actually forgot to do that; latest kotest wanted 1.6. Can still downgrade to 1.5.0 if you want?
But since I didn't have problems with 1.6, what problems do you think there can be with 1.5?

(I don't remember running into any myself)

Reply:
I'm perfectly fine with 1.6. Just don't want to break something. But in the worse case, in the future it can be rolled back to 1.5.
Let's leave it as it is. :)